org-code-javabuilder/media/src/test/java/org/code/media/ColorTest.java

```diff
@@ -12,11 +12,38 @@ public void defaultAlphaIs255() {
   }
 
   @Test
-  public void resettingColorSetsAlpha() {
-    Color c = new Color(0, 0, 0, 0);
-    assertEquals(0, c.getAlpha());
-    c.setBlue(255);
-    // now alpha should be 255 (max opacity), since we set the blue value on the color
-    assertEquals(255, c.getAlpha());
+  public void testCopyWithRedCreatesCopyWithNewRedValue() {
+    final int red = 100;
+    final int green = 150;
+    final int blue = 200;
+    final int newRed = 50;
+    final Color copied = Color.copyWithRed(new Color(red, green, blue), newRed);
+    assertEquals(newRed, copied.getRed());
+    assertEquals(green, copied.getGreen());
+    assertEquals(blue, copied.getBlue());
```

Comment:
can you also add a check that `copied` does not equal (strict equality) the original color?

Reply:
Sure, done!